rm_vk/rm_vk/Core/NetworkService/Core.swift

```diff
@@ -0,0 +1,21 @@
+// Core.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Alamofire
+
+/// Сетевые запросы
+final class Core {
```

Comment:
```suggestion
final class NetworkCoreService {
```
Не понятно что за ядро (Core)